deployment/src/main/resources/templates/api.qute

```diff
@@ -65,7 +65,7 @@ public interface {classname} {
     {#if op.hasProduces}
     @Produces(\{{#for produce in op.produces}"{produce.mediaType}"{#if produce_hasNext}, {/if}{/for}\})
     {/if}
-    @GeneratedMethod ("{op.operationId}")
+    @GeneratedMethod ("{op.operationIdOriginal}")
```

Comment:
is it set by openapi?

Reply:
yep

Reply:
I see, thanks.
What are the possible issues to use only the Original value instead the operationID?

Reply:
@spolti it is the original value of the _operationId_ :) instead of the a sanitized version of it.
Without this fix, an `operationId:Jobs doJob` would result in `jobsDoJob` when we fact, we just want it to add `Jobs doJob`, which is the original value.